.github/actions/github-release/main.js

```diff
@@ -72,15 +76,28 @@ async function run() {
   });
 
   // Upload all the relevant assets for this release as just general blobs.
+  const retries = 10;
   for (const file of glob.sync(files)) {
     const size = fs.statSync(file).size;
     core.info(`upload ${file}`);
-    await octokit.repos.uploadReleaseAsset({
-      file: fs.createReadStream(file),
-      headers: { 'content-length': size, 'content-type': 'application/octet-stream' },
-      name: path.basename(file),
-      url: release.data.upload_url,
-    })
+    for (let i = 0; i < retries; i++) {
+      try {
+        await octokit.repos.uploadReleaseAsset({
+          data: fs.createReadStream(file),
+          headers: { 'content-length': size, 'content-type': 'application/octet-stream' },
+          name: path.basename(file),
+          url: release.data.upload_url,
+        });
+        break;
+      } catch (e) {
+        if (i === retries - 1)
+          throw e;
+        console.log("ERROR: ", JSON.stringify(e, null, 2));
+        console.log("ERROR: ", e.message);
+        console.log(e.stack);
+        console.log("RETRYING after 10s");
+        await sleep(10000)
```

Comment:
I guess randomized exponential backoff doesn't really matter in this case, but little lights go off in my head telling me to do that everytime there is a retry loop :-P

Reply:
Heh I agree, I'm hoping a pretty long sleep here is "roughly good enough"